iphone/Maps/UI/DocumentPicker/FileType.swift

```diff
@@ -0,0 +1,22 @@
+import UniformTypeIdentifiers
+
+// TODO: Remove this type-wrapper and use custom UTTypeIdentifier that is registered into the Info.plist after updating to the iOS >= 14.0.
+struct FileType {
+  let fileExtension: String
+  let typeIdentifier: String
+}
+
+extension FileType {
+  static let kml = FileType(fileExtension: "kml", typeIdentifier: "com.google.earth.kml")
+  static let kmz = FileType(fileExtension: "kmz", typeIdentifier: "com.google.earth.kmz")
+  static let gpx = FileType(fileExtension: "gpx", typeIdentifier: "com.topografix.gpx")
```

Comment:
OM also supports importing kmb - mapsme binary files.

Reply:
@kirylkaveryn is it possible to register this file type, and test if it works?

Reply:
@biodranik how can I get/create the kmb file?

Reply:
Yes, please create an issue. Try to extract kmb files from Maps.Me's bookmarks folder using file sharing, and let me know if it won't work.